client/src/editorWebviewScript.ts

```diff
@@ -82,13 +88,21 @@ const machine = createMachine<WebViewMachineContext, EditorWebviewScriptEvent>(
       index: 0,
       layoutString: undefined,
       token: undefined,
+      implementations: {
+        implementations: {
```

Comment:
Same question as https://github.com/statelyai/registry/pull/505/files#r816433023 - why is `implementations` here twice?

Reply:
Same answer, will refactor away